packages/kbn-unified-field-list/src/containers/unified_field_list_item/field_list_item.tsx

```diff
@@ -15,6 +15,8 @@ import type { FieldsMetadataPublicStart } from '@kbn/fields-metadata-plugin/publ
 import { Draggable } from '@kbn/dom-drag-drop';
 import type { DataView, DataViewField } from '@kbn/data-views-plugin/public';
 import { Filter } from '@kbn/es-query';
+import { fieldSupportsBreakdown } from '@kbn/field-utils';
+import { isESQLFieldGroupable } from '@kbn/esql-utils/src/utils/esql_fields_utils';
```

Comment:
Add the `isESQLFieldGroupable ` into the index files of the esql-utils package to import from `@kbn/esql-utils` 

We are doing the same for each function and I want to keep the same format